actors/builtin/market/market_actor.go

```diff
@@ -270,14 +296,9 @@ func (a Actor) VerifyDealsOnSectorProveCommit(rt Runtime, params *VerifyDealsOnS
 	store := adt.AsStore(rt)
 
 	// Update deal states.
-	dealWeight := big.Zero()
-	verifiedWeight := big.Zero()
-	var err error
 	rt.State().Transaction(&st, func() interface{} {
-		dealWeight, verifiedWeight, err = ValidateDealsForActivation(&st, store, params.DealIDs, minerAddr, params.SectorExpiry, currEpoch)
-		if err != nil {
-			rt.Abortf(exitcode.ErrIllegalState, "failed to validate proposals for activation: %v", err)
-		}
+		_, _, err := ValidateDealsForActivation(&st, store, params.DealIDs, minerAddr, params.SectorExpiry, currEpoch)
+		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to validate proposals for activation")
```

Comment:
I think we can probably remove this. Leave it for now, I'll file an issue to discuss.